receivers/email/email.go

```diff
@@ -108,3 +112,21 @@ func (en *Notifier) Notify(ctx context.Context, alerts ...*types.Alert) (bool, e
 func (en *Notifier) SendResolved() bool {
 	return !en.GetDisableResolveMessage()
 }
+
+func readFile(path string) ([]byte, error) {
+	_, err := os.Stat(path)
+	if err != nil {
+		return nil, err
+	}
+
+	f, err := os.Open(path)
+	if err != nil {
+		return nil, err
+	}
+
+	defer func() {
+		_ = f.Close()
+	}()
+
+	return io.ReadAll(f)
```

Comment:
WDYT about checking file size and creating a hard limit for the size of file? We can take double of what render service produces.

Reply:
This function is going away in the next PR, it's just a stop-gap. So, we can discuss adding something like this in the next one IMO.